package.json

```diff
@@ -43,17 +43,23 @@
     "clean": "node scripts/clean-modules/index.js",
     "clean:install": "npm run clean && npm install",
     "clean:obsolete-snapshots": "npm test -- -u",
+    "compile:heroku": "webpack --config src/config/webpack.prod.config -p",
+    "compile:prod": "webpack --config src/config/webpack.prod.config -p",
     "danger": "danger",
+    "heroku-postbuild": "npm install --only=dev && npm run compile:heroku",
```

Comment:
Could we just use `compile:prod` for `heroku-postbuild`? `compile:prod` and `compile:heroku` are the same script

Reply:
Seems reasonable. We could update this in the mono-repos too.